libraries/c_sdk/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -754,6 +837,342 @@ static IotMqttError_t _subscriptionCommon( IotMqttOperationType_t operation,
     IOT_FUNCTION_CLEANUP_END();
 }
 
+/*-----------------------------------------------------------*/
+/* Connect Serialize Wrapper*/
+static IotMqttError_t _connectSerializeWrapper( const IotMqttConnectInfo_t * pConnectInfo,
+                                                uint8_t ** pConnectPacket,
+                                                size_t * pPacketSize )
+{
+    IotMqttError_t status = IOT_MQTT_SUCCESS;
+    size_t remainingLength = 0UL;
+    MQTTConnectInfo_t connectInfo;
+
+    connectInfo.cleanSession = pConnectInfo->cleanSession;
```

Comment:
Don't we need to assert for validating parameters?

Reply:
They are getting validated in the IOT_mqttConnect().After validating only ,I am using them in this serialization function.